docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>The description is very well done in general. The only nitpick is that this is not showing the response, but the mean of the response. Since both are in the same scale the difference is subtle.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>